lib/utils/cookie.js

```diff
@@ -33,19 +34,19 @@ cookie.prototype.get = function(str){
 };
 
 cookie.prototype.set = function(str, value){
```

Comment:
`js-cookies` supports a third `expire` parameter. Could we support it in this `set` method?